lua/xadmin/_config/sh_config.lua

```diff
@@ -33,33 +33,90 @@ xAdmin.Config.Admin = 80 -- Admin
 -- The power level that can see admin chat
 xAdmin.Config.AdminChat = 30
 -- The color of admin chat
-xAdmin.Config.AdminChatColor = Color(0,150,255)
--- Color of the logs
+xAdmin.Config.AdminChatColor = Color(255,255,255)
+-- Admin Chat Prefix Color
+xAdmin.Config.AdminChatPrefixColor = Color(46, 170, 200)
+-- Admin Chat Prefix
+xAdmin.Config.AdminChatPrefixText = "[Admin Chat] "
+-- Color of the logs prefix
 xAdmin.Config.ColorLog = Color(46, 170, 200)
+-- Color of the logs (not the prefix)
+xAdmin.Config.ColorLogText = Color(255, 255, 255)
+-- Color of the console if console does something
+xAdmin.Config.ColorConsole = Color(255, 0, 0)
+-- Logs Prefix
+xAdmin.Config.LogPrefix = "| "
 -- Prop limit function
 xAdmin.Config.PropLimit = true
+xAdmin.Config.PropLimitNotify = true
 xAdmin.Config.DefaultPropLimit = 25
+xAdmin.Config.PropLimitInfinText = "Infinite" -- If you have a prop limit of -1 (infinite) than the prop limit will display in the message as this.
+-- Dont allow people who are frozen to spawn props
+xAdmin.Config.DisallowFrozenPropSpawning = true
 
 -- The table prefix used when making the unique tables. If you're running multiple servers this should be unique else they will share data
 xAdmin.Config.Name = "svr1"
 -- The unique name of this server. This will be used as a unique identifier in the bans archive
 xAdmin.Config.FullName = "Server 1"
 
+-- Custom Concommands
+-- For instance, putting "ulx" in here will let people use 'ulx slay lion' in console
+-- even though we are using xAdmin (hopefully)
+-- THIS MIGHT CAUSE ERRORS WITH OTHER ADDONS
+xAdmin.Config.CustomConsoleCommands = {
+	["ulx"] = true,
+	["sam"] = true,
+}
+
+/*=============
+Physgun Config
+=============*/
+-- Master switch to disable / enable everything with Physgun in xAdmin
+-- You are required to have this true even if you only have one of the
+-- two features enabled!
+xAdmin.Config.EnablePhysgunFeatures = true
+
+-- Allow xAdmin to control staff picking people up with their physgun
+xAdmin.Config.PhysgunEnablePeople = true
+-- Power level that is allowed to pick people up with their physgun
+xAdmin.Config.PhysgunPickupPlayerPowerlevel = 30
+
+-- Allow xAdmin to control staff picking vehicles up with their physgun
+xAdmin.Config.PhysgunEnableVehicle = true
+-- Power level that is allowed to pick vehicles up with their physgun
+xAdmin.Config.PhysgunPickupVehiclePowerlevel = 40
+
 /*===============
 Formatting Config
 =================*/
 
 -- The chat prefix
 xAdmin.Config.Prefix = "!"
 
--- The ban message
--- The %s formatting order is: Banning admin, time left, ban reason
-xAdmin.Config.BanFormat = [[------------
+-- Admin Chat Prefix
+xAdmin.Config.AdminChatPrefix = "@"
+
+-- PermaBan Length
+xAdmin.Config.StrForPermBan = "Forever"
+
+-- Enable the noclip warning
+xAdmin.Config.EnableNoclipWarning = true
 
--- Banned --
-Banned by: %s
-Time left: %s
-Reason: %s
+-- Warning sent in chat when running "noclip" concommand
+xAdmin.Config.NoclipWarning = {
```

Comment:
You can actually pass "\n" as a line break for chat messages, tho it'll look like [this](https://i.notgyazo.club/PdqCPg1Zp.png). So I understand why that might not be visually appealing